Dockerfile

```diff
@@ -7,6 +7,10 @@ RUN apt-get update && apt-get install -y \
     --no-install-recommends && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install -y \
+gpg
```

Comment:
Please install gpg with the install step before, saving time and layer size!

Reply:
ups sorry it was for testing locally :-)